### PR TITLE
Housekeeping

### PR DESCRIPTION
**Pull request by GwendolenLynch**
@ro0NL please feel free to ask for any/all of this to be removed/changed.

The initial use case for this PR was tracking down root cause on some things, and having the imports unresolvable made things a bit tricky to eyeball :smile: 

I figured I'd add in some base CS functionality while here.

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -19,8 +19,16 @@
         "msgphp/user": "^1.0",
         "symfony/config": "^3.4|^4.0",
         "symfony/dependency-injection": "^3.4|^4.0",
+        "symfony/framework-bundle": "^3.4|^4.0",
```
Comment (ro0NL) [inline composer.json]: same, can be removed right?

Comment (GwendolenLynch) [reply composer.json]: It is used [here](https://github.com/msgphp/user-bundle/blob/master/src/DependencyInjection/Extension.php#L59) … but totally up to you.

Comment (ro0NL) [reply composer.json]: ok. require-dev then?

Comment (GwendolenLynch) [reply composer.json]: `<insert facepalm emoji here>` … I blame Christmas :rofl:

**Review by ro0NL — APPROVED** [review]

**Pull request merged**